components/drop-zone/provider.js

```diff
@@ -68,7 +68,7 @@ class DropZoneProvider extends Component {
 		} );
 
 		this.dropzones.forEach( ( { updateState } ) => {
-			updateState( { isDraggingOverDocument, isDraggingOverElement: false, position: null } );
+			updateState( { isDraggingOverDocument: false, isDraggingOverElement: false, position: null } );
```

Comment:
Should consider splitting long line across multiple.

Reply:
we definitely need prettier 👍 one hour dev time gained per day :)

Reply:
[`object-curly-newline`](https://eslint.org/docs/rules/object-curly-newline) has options for configuring number of properties before newlines are required, **and** includes a fixer!

Reply:
Do you use the auto-fixer on IDE save? I never saw ESlint used for this, but yeah could be a good alternative for now 👍.

I'm using prettier "auto-save format" in other projects and it's really a huge gain.

Reply:
I found a similar config in VS Core, I'll be trying it for some days. I'm not adding the rule right now, but we should consider it in a separate PR.